README.md

```diff
@@ -13,14 +13,14 @@ Certificate Authority Service (CAS)](https://cloud.google.com/certificate-author
 Enable the Certificate Authority API (`privateca.googleapis.com`) in your GCP project by following the
 [official documentation](https://cloud.google.com/certificate-authority-service/docs/quickstart).
 
-#### CAS-managed CAs
+#### CAS-managed Certificate Authorities
 
 You can create a root certificate authority as well as an intermediate
 certificate authority ("subordinate") in your current Google project with:
 
 ```sh
-gcloud beta privateca roots create my-ca --subject="CN=root,O=my-ca"
-gcloud beta privateca subordinates create my-sub-ca  --issuer=my-ca --location us-east1 --subject="CN=intermediate,O=my-ca,OU=my-sub-ca"
+gcloud beta privateca roots create my-ca --subject="CN=root,O=my-ca" --location us-east1 --max-chain-length=1
```

Comment:
I believe that max-chain-length as 1 is correct here since it's what the UI sets by default.